implementations/rust/ockam/ockam_identity/src/service/models.rs

```diff
@@ -0,0 +1,241 @@
+#![allow(missing_docs)]
+
+use minicbor::{Decode, Encode};
+use ockam_api::{CowBytes, CowStr};
+
+#[cfg(feature = "tag")]
+use ockam_core::TypeTag;
+
+#[derive(Debug, Clone, Encode, Decode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct CreateResponse<'a> {
```

Comment:
Is there `CreateRequest`?
In the API doc there is a `name` field.

Reply:
What API doc? We don't have names for Identities

Reply:
In the future we definitely want to at least support providing `key_id`, for now it should be fine without it